src/components/pages/PendingApplications/MentorModal.js

```diff
@@ -0,0 +1,110 @@
+import { List, Tag } from 'antd';
+import '../../../styles/styles.css';
+import './PendingApplication.css';
+
+const MentorModal = ({ applicant }) => {
+  return (
+    <>
+      <div className="profile-intro">
+        <div className="image-container">
+          <img
+            src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
+            alt="I can do this if I work hard enough and practice my coding skills"
+          />
+        </div>
+        <div className="profile-intro-description">
+          <h2>
+            {applicant.first_name} {applicant.last_name}
+          </h2>
+          <div className="preferences">
+            <div>
+              <p>Can mentor in: </p>
+            </div>
+            <div className="tags-container">
+              {applicant.tech_stack.map(subject => {
+                return (
+                  <div className="mentor-tag" key={subject}>
+                    <Tag color={'green'}>{subject}</Tag>
+                  </div>
+                );
+              })}
+            </div>
+          </div>
+        </div>
+      </div>
+      <List size="small">
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Role</p>
+          </div>
+          <div className="list-item-column">
+            <p>
+              {applicant.role_name[0].toUpperCase() +
```

Comment:
Depending how often you have to do this, this might make more sense as a utility function that's imported (ex. `capitalizeFirstLetter(applicant.role_name)`)